packages/block-editor/src/components/block-switcher/block-transformations-menu.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { MenuGroup, MenuItem } from '@wordpress/components';
+import { getBlockMenuDefaultClassName } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import BlockIcon from '../block-icon';
+
+const BlockTransformationsMenu = ( {
+	className,
+	possibleBlockTransformations,
+	onSelect,
+} ) => {
+	return (
+		<MenuGroup label={ __( 'Transform to' ) } className={ className }>
+			{ possibleBlockTransformations.map( ( item ) => {
+				const { name, icon, title } = item;
+				return (
+					<MenuItem
+						key={ name }
+						className={ getBlockMenuDefaultClassName( name ) }
```

Comment:
Is this class necessary here?

Reply:
This class is used in `BlockTypesList` ( it was used before ). I came across some failing e2e tests that were using the `transform` functionality and this class seemed a good idea to preserve, as it might be used by other e2e tests in the future.

Reply:
I don’t think that e2e test should shape production code, and I’m sure you can use accessible labels to achieve the same result as block names are unique. It isn’t a big issue but if we can remove unused code with little effort we should always aim for it.

Reply:
>I don’t think that e2e test should shape production code

I definitely agree with this!

In this specific case though ( `MenuItem` ) there is not any accessible labels to match, besides the content itself. `MenuItem` has an [info property](https://developer.wordpress.org/block-editor/components/menu-item/#info) but this would end up adding stuff again.

So, would it be preferable to search through the content?

Reply:
Well, the content is the accessible label, this is what is going to be announced when users depend on screen readers. By using this approach, we also ensure that UI is accessible. It's harder to target elements by content, but there are many existing tests that do it.